fourstoryFlask/routes.py

```diff
@@ -49,4 +49,6 @@ def history(date):
     checkins = requests.get(f'https://api.foursquare.com/v2/users/self/checkins?oauth_token={token}&v=20190101&beforeTimestamp={before_timestamp}&afterTimestamp={after_timestamp}')
     prevDay = date_obj - timedelta(days=1)
     prevDayStr = prevDay.strftime('%Y-%m-%d')
-    return render_template("daily-checkins.html", checkins=checkins.json(), date=date_str, prevDayStr=prevDayStr)
\ No newline at end of file
+    nextDay = date_obj + timedelta(days=1)
```

Comment:
python generally prefers `snake_case` (see PEP8 / "black", the equivalent of prettier)